### DISCUSSION TITLE
About contributions and local build: not finding parent pom

### DISCUSSION THREAD

**Discussion by LucasJC** [awspring/spring-cloud-aws#973] [Q&A] [answered]
Hey guys! I cloned the main branch and was setting up my local environment but I couldnt get it to build as is.
The error I got was that the parent pom spring-cloud-build was not available in the version that is currently declared (4.1.0-RC1).

Looking at the spring-cloud-build repo I see the current work is on 4.1.0-SNAPSHOT. 
Am I missing something? I cant seem to find where this RC1 build should be. 

PS: I could build the project changing to a publicly available version e.g. 4.0.0 but I dont feel this is the right way to work on contributions.

**Comment by maciejwalkowiak** [comment, marked as answer]
Hi LucasJC! You must enable "spring" Maven profile to build against Spring Cloud's snapshots or RCs.

In IDEA:

![image](https://github.com/awspring/spring-cloud-aws/assets/1357927/93f4ea02-db34-41ee-9dde-3920b229c00b)

In command line:

```console
./mvnw install -Pspring
```

**Reply by LucasJC** [reply]
Thanks a lot for your help Maciel! That worked perfectly

**Reply by maciejwalkowiak** [reply]
Great to hear 🙂
